pkg/trace/api/api.go

```diff
@@ -28,6 +28,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/appsec"
```

Comment:
This import is in the wrong group of imports.

Reply:
Fixed